### DISCUSSION TITLE
tampilan website pada saat di desktop dengan mobile bisa berbeda?

### DISCUSSION THREAD

**Discussion by ayambakarjevry** [deaafrizal/forum#84] [Troubleshooting Room]
mau tanya dong bg de. kenapa pas masuk ke website seseorang menggunakan desktop / komputer bentuk nya A , tetapi kalau kita masuk ke website yang sama melalui seluller tampilannya berubah drastis. kenapa ya bg de

**Comment by mwafa** [comment]
namanya responsive,

dia bisa berdasarkan lebar dari layarnya untuk banyak web modern

tapi untuk web klasik ada yang pakai user agent untuk deteksi apakah dia mobile.
